README.md

```diff
@@ -104,12 +113,12 @@ Write this grammar using EBNF with common extensions
   You can hand draw it and snap a picture, or use a drawing tool. Either way,
   upload the image, and put the file name in the answer below.
 
-  (Hint: my answer has seven states including the start and end states)
+  (Hint: my answer has six states including the start and end states)
```

Comment:
# 5

I took one off because S2 is unnecessary (S1 can loop back to itself when it gets an adjective).